content/en/docs/collector/getting-started.md

```diff
@@ -1,358 +1,90 @@
-apk add --allow-untrusted otelcol_{{% param collectorVersion %}}_linux_amd64.apk
-```
-
-{{% /tab %}} {{% tab ARM64 %}}
-
-```sh
-apk update
-apk add wget shadow
-wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_arm64.apk
-apk add --allow-untrusted otelcol_{{% param collectorVersion %}}_linux_arm64.apk
-```
-
-{{% /tab %}} {{% tab i386 %}}
-
-```sh
-apk update
-apk add wget shadow
-wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_386.apk
-apk add --allow-untrusted otelcol_{{% param collectorVersion %}}_linux_386.apk
-```
-
-{{% /tab %}} {{< /tabpane >}}
-
-### DEB Installation
-
-To get started on Debian systems run the following replacing
-`v{{% param collectorVersion %}}` with the version of the Collector you wish to
-run and `amd64` with the appropriate architecture.
-
-{{< tabpane text=true >}} {{% tab AMD64 %}}
-
-```sh
-sudo apt-get update
-sudo apt-get -y install wget systemctl
-wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_amd64.deb
-sudo dpkg -i otelcol_{{% param collectorVersion %}}_linux_amd64.deb
-```
+To follow this tutorial you need the following Go 1.20 or higher.
 
-{{% /tab %}} {{% tab ARM64 %}}
+## Test the OpenTelemetry Collector in five minutes
 
-```sh
-sudo apt-get update
-sudo apt-get -y install wget systemctl
-wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_arm64.deb
-sudo dpkg -i otelcol_{{% param collectorVersion %}}_linux_arm64.deb
-```
+1. [Download and install](/docs/collector/installation) the Collector for your
+   operating system and architecture. For example:
 
-{{% /tab %}} {{% tab i386 %}}
+   ```sh
+   sudo apt-get update
+   sudo apt-get -y install wget systemctl
+   wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_amd64.deb
+   sudo dpkg -i otelcol_{{% param collectorVersion %}}_linux_amd64.deb
+   ```
 
-```sh
-sudo apt-get update
-sudo apt-get -y install wget systemctl
-wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_386.deb
-sudo dpkg -i otelcol_{{% param collectorVersion %}}_linux_386.deb
-```
+   Alternatively, you can download and run the Docker container:
 
-{{% /tab %}} {{< /tabpane >}}
+   ```sh
+   docker pull otel/opentelemetry-collector-contrib:{{% param collectorVersion %}}
+   docker run otel/opentelemetry-collector-contrib:{{% param collectorVersion %}}
+   ```
```

Comment:
Let's keep it simple here and tell people that for the sake of the getting started they should use docker and they will find different deployment patterns on that other page.

Note: let's not use contrib here, I don't think you will make use of any additional feature of it?

Reply:
Also, add some notes on what they should see on their console

Reply:
Solved in https://github.com/open-telemetry/opentelemetry.io/pull/3412/commits/5826d97fc3f8b83911360fa7fa850fa91f1576b1

Note that at least two ports must be exposed when running the Docker container